src/timetable/types.ts

```diff
@@ -1,6 +1,17 @@
 import { PopoverV2TriggerType } from "../popover-v2";
+import { RowBarAlternateColors, RowBarMainColors } from "./const";
 
-export type TimeTableCellType = "filled" | "blocked" | "default" | "disabled";
+export type TimeTableCellType =
+    | "filled"
+    | "blocked"
+    | "default"
+    | "disabled"
+    | "pending";
+
+export interface RowBarColors {
```

Comment:
I don't think this is meant to be exposed as a public typing 👀 you could introduce a `internal-types.ts` file to store this